superset/tags/schemas.py

```diff
@@ -58,7 +58,7 @@ class TaggedObjectEntityResponseSchema(Schema):
 
 
 class TagObjectSchema(Schema):
-    name = fields.String()
+    name = fields.String(validate=Length(min=1))
```

Comment:
Does this logic also need to reside within the Command or DAO? Additionally do we have any tests for schema validation?

Reply:
it lies in the API, I'm working on a test now to validate